### PR TITLE
fixed help messages to be more clear

### PR DESCRIPTION
**Pull request by mdesco**
Quick fix

### CODE REVIEW

**Comment by scil-admin** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_visualize_connectivity.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 - Coordinates (0..N)
 - Labels (using --labels_list)
 - Names (using --labels_list and --lookup_table)
+Examples of labels_list.txt and lookup_table.json can be found in the
```
Comment (arnaudbore) [inline scripts/scil_visualize_connectivity.py:13]:
```suggestion
Examples of labels_list.txt and lookup_table.json can be found in the
```

File: scripts/scil_visualize_connectivity.py [hunk 2]
```diff
@@ -10,6 +10,8 @@
 - Coordinates (0..N)
 - Labels (using --labels_list)
 - Names (using --labels_list and --lookup_table)
+Examples of labels_list.txt and lookup_table.json can be found in the 
+freesurfer_flow output (https://github.com/scilus/freesurfer_flow)
```
Comment (arnaudbore) [inline scripts/scil_visualize_connectivity.py:14]:
```suggestion
freesurfer_flow output (https://github.com/scilus/freesurfer_flow)
```

**Review by arnaudbore — APPROVED** [review]
LGTM

**Pull request merged**